src/Disks/VolumeJBOD.cpp

```diff
@@ -53,6 +53,18 @@ VolumeJBOD::VolumeJBOD(
     static constexpr UInt64 MIN_PART_SIZE = 8u * 1024u * 1024u;
     if (max_data_part_size != 0 && max_data_part_size < MIN_PART_SIZE)
         LOG_WARNING(logger, "Volume {} max_data_part_size is too low ({} < {})", backQuote(name), ReadableSize(max_data_part_size), ReadableSize(MIN_PART_SIZE));
+
+    are_merges_allowed = !config.getBool(config_prefix + ".disable_merges", false);
```

Comment:
By the way AFAICS right now cross volume/disk merges are allowed, how about adding some option to prohibit only them? (`disable_cross_merges`, or similar, in addition to `disable_merges`)

Reply:
@alz @filimonov What do you think?

Reply:
1) Forbid cross volume merges - indeed, sounds as a good idea.
2) current solution doesn't forbid that? 
3) cross disk of same volume - see no reason why should be ever forbidden

Reply:
We can get back to that subject later, if it will be needed.

For now we only need to forbid merges happening on very slow network storages / storages with some extra costs per every access. That goal is solved by that PR.